src/main/scala/mesosphere/marathon/core/event/Events.scala

```diff
@@ -213,13 +213,23 @@ case class MesosStatusUpdateEvent(
 
 case class InstanceChanged(
     id: Instance.Id,
+    runSpecVersion: Timestamp,
```

Comment:
I know this is a little odd, but it'd be great if we used offsetdatetime in more places. I'd eventually like to outright replace it and exclusively use Java 8 time as it is superior and effectively joda time 2